app/models/form_submission_attempt.rb

```diff
@@ -25,14 +25,21 @@ class FormSubmissionAttempt < ApplicationRecord
 
     event :fail do
       after do
+        log_info = { form_submission_id:,
+                     benefits_intake_uuid: form_submission&.benefits_intake_uuid,
+                     form_type: form_submission&.form_type }
         if should_send_simple_forms_email
-          Rails.logger.info({
-                              message: 'Preparing to send Form Submission Attempt error email',
-                              form_submission_id:,
-                              benefits_intake_uuid: form_submission.benefits_intake_uuid,
-                              form_type: form_submission.form_type
-                            })
+          Rails.logger.info('Preparing to send Form Submission Attempt error email', log_info)
           simple_forms_enqueue_result_email(:error)
+        else
+          # Do not love hard coding this in here like this.
+          # Hoping to refactor this at somepoint to better support various emailing on arbitrary or inherited classes
+          should_send_form526_form4142_email
```

Comment:
As is, this will just be a boolean, right? It won't do anything.

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/18801/files#diff-89e540b125e73301d05a170e3451dabddda1b36524d476e8bdbb31751efea812R98

Yep, line 98

Reply:
oh the commenting messed this up, this is supposed to be on the previous line and yes this should be an elsif